Kubernetes/omsagent.yaml

```diff
@@ -571,6 +571,103 @@ spec:
           configMap:
             name: container-azm-ms-agentconfig
             optional: true
+---
+  apiVersion: apps/v1
+  kind: DaemonSet
+  metadata:
+   name: omsagent-win
+   namespace: kube-system
+   labels:
+    component: oms-agent
+    tier: node
+  spec:
+   updateStrategy:
+    type: RollingUpdate
+   selector:
+    matchLabels:
+      component: oms-agent
+      tier: node
+   template:
+    metadata:
+      labels:
+        component: oms-agent
+        tier: node
+      annotations:
+        agentVersion: "1.10.0.1"
+        dockerProviderVersion: "9.0.0-1"
+        schema-versions: "v1"
+    spec:
+     serviceAccountName: omsagent
+     containers:
+       - name: omsagent-win
+         image:  mcr.microsoft.com/azuremonitor/containerinsights/cidev:win-20200507-062708z-7f95982
```

Comment:
I persume, you guys will be fixing the image format.

Reply:
yes, this will be part of the release task (when we are making the changes in the aks rp)